### PR TITLE
[iOS] Add empty search result

### LINKED ISSUE #607: [iOS] Add empty view for search result
**Idea Description**

- Please add empty view for search result.

**Reference images and links**
スクリーンショット 2022-09-19 13 02 02

### PR DESCRIPTION
**Pull request by NakaokaRei**
## Issue
- close #607

## Overview (Required)
- added empty view for search result.

## Screenshot
Before | After
:--: | :--:
Simulator Screen Shot - iPhone 14 Pro - 2022-09-20 at 18 21 18 | Simulator Screen Shot - iPhone 14 Pro - 2022-09-20 at 18 18 21

### CODE REVIEW

File: app-ios/Sources/SearchFeature/EmptyResultView.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+public struct EmptyResultView: View {
+
+    public init() {}
+
+    public var body: some View {
+        VStack {
+            Assets.search.swiftUIImage
+                .resizable()
+                .scaledToFit()
+                .frame(width: 58.3, height: 58.3)
```
Comment (ry-itto) [inline app-ios/Sources/SearchFeature/EmptyResultView.swift:14]: Please use 80x80 size!
Vector size is 58.3, but icon size is 80x80 🙏🏼 
スクリーンショット 2022-09-20 18 24 42

File: app-ios/Sources/SearchFeature/SearchView.swift [hunk 2]
```diff
@@ -87,17 +87,25 @@ public struct SearchView: View {
...
-                                    isFavorite: timetableItem.isFavorited,
-                                    searchText: viewStore.searchText
-                                ) { id, isFavorited in
-                                    viewStore.send(.setFavorite(id, isFavorited))
+                Group {
+                    if viewStore.searchResult.values.allSatisfy { timetable in
+                        timetable.timetableItems.isEmpty
+                    } {
+                        EmptyResultView()
+                    } else {
```
Comment (ry-itto) [inline app-ios/Sources/SearchFeature/SearchView.swift:95]: How about this? 👀 
```suggestion
                    if viewStore.searchResult.values.allSatisfy(\.timetableItems.isEmpty) {
                        EmptyResultView()
                    } else {
```

Comment (NakaokaRei) [reply app-ios/Sources/SearchFeature/SearchView.swift:95]: @ry-itto 
This one is simpler.
Thanks!

File: app-ios/Sources/SearchFeature/EmptyResultView.swift (hunk 1 above)
Comment (NakaokaRei) [reply app-ios/Sources/SearchFeature/EmptyResultView.swift]: @ry-itto 
I fixed at https://github.com/DroidKaigi/conference-app-2022/pull/632/commits/678ea2dce845470205e70398f981ec56efbc8fe5 !

**Review by ry-itto — APPROVED** [review]
LGTM! Thanks!!

**Pull request merged**